docs/zh/latest/plugins/consumer-restriction.md

```diff
@@ -117,6 +119,74 @@ HTTP/1.1 403 Forbidden
 {"message":"The consumer_name is forbidden."}
 ```
 
+### 如何限制 `allowed_by_methods`
+
+下面是一个示例，在指定的route上开启了`consumer-restriction`插件，限制`jack1`只能使用`POST`进行访问：
```

Comment:
```suggestion
下面是一个示例，在指定的 route 上开启了`consumer-restriction`插件，限制`jack1`只能使用`POST`进行访问：
```
We should add a space between Chinese and English. You can ref https://github.com/sparanoid/chinese-copywriting-guidelines